ratpack-core/src/main/java/ratpack/http/client/internal/RequestSpecBacking.java

```diff
@@ -51,6 +52,7 @@ public RequestSpecBacking(MutableHeaders headers, URI uri, ByteBufAllocator byte
     this.byteBufAllocator = byteBufAllocator;
     this.requestParams = requestParams;
     this.bodyByteBuf = byteBufAllocator.buffer(0, 0);
+    this.decompressResponse = false;
```

Comment:
That breaks backwards compatibility and a few tests, so I wanted to avoid it :stuck_out_tongue:
If it's not a big concern to you (understandably, it's not even 1.0 at this point) let me know.